charts/prometheus-pingdom-exporter/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: prometheus-pingdom-exporter
-version: 3.0.3
+version: 3.0.4
```

new feature should raise minor version.

```suggestion
version: 3.1.0
```